packages/plugins/apollo-tracing/src/index.ts

```diff
@@ -39,10 +39,10 @@ type TracingContextObject = {
   hrtime: [number, number];
 };
 
-export const useApolloTracing = (): Plugin => {
+export const useApolloTracing = (): Plugin<{}, { [apolloTracingSymbol]: TracingContextObject }> => {
```

Comment:
`Record<string, any>` is assignable to `{}`, as `{}` matches basically any object. There is no concept of "exact" types in TypeScript.

Correct me if I am wrong, but it seems like you are expecting `{}` to behave like `Record<string, never>`.

See: https://www.typescriptlang.org/play?#code/MYewdgzgLgBAhjAvDAFASzABwK5QFwwDeAvgJR4BuIaAJkgHxHEBQoksARkqhjvjACUApqABONADzRRGAOYwAPjAgBPALYcQAGwA0MMEIpDR9cjCq0GTZq3DQYAMxAgCwsZOlzFy9Zt3wwFUZkEhs4FCcQUmYOCOdooA

Reply:
I think I am vaguely referring to an eslint rule that people might use in their project. But I think you are right, I am not sure it would be a problem! 

```
Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.eslint[@typescript-eslint/ban-types](https://github.com/typescript-eslint/typescript-eslint/blob/v4.22.0/packages/eslint-plugin/docs/rules/ban-types.md)
```